tests/system/I18n/TimeDifferenceTest.php

```diff
@@ -194,12 +202,25 @@ public function testHumanizeNoDifference()
 		$this->assertEquals('Just now', $diff->humanize('en'));
 	}
 
-	public function testGetter()
+	public function testGetterUTC()
+	{
+		$current = Time::parse('March 10, 2017', 'UTC');
+		$diff    = $current->difference('March 18, 2017', 'UTC');
+
+		$this->assertEquals(8, $diff->getDays());
+		$this->assertEquals(8, $diff->days);
+		$this->assertEquals(-8, (int) round($diff->getDays(true)));
+		$this->assertNull($diff->nonsense);
+	}
+
+	public function testGetterChicagoTime()
 	{
 		$current = Time::parse('March 10, 2017', 'America/Chicago');
 		$diff    = $current->difference('March 18, 2017', 'America/Chicago');
 
-		$this->assertEquals(-8, (int) round($diff->days));
+		$this->assertEquals(7, $diff->getDays());
+		$this->assertEquals(7, $diff->days);
+		$this->assertEquals(-8, (int) round($diff->getDays(true)));
```

Comment:
I don't know why the difference is 7 days in Chicago time.

Reply:
It is probably because of summer time.

Reply:
For example, DateTime::diff() does not have this problem.